xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.35.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?oxygen RNGSchema="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" type="xml"?><TEI xmlns:xi="http://www.w3.org/2001/XInclude" xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader><!-- testing1 -->
+      <fileDesc>
+         <titleStmt>
+            <title>IN.Bloom.IUAM.G.63.105.35</title>
+         </titleStmt>
+         <publicationStmt>
+            <xi:include  href="http://library.brown.edu/usep_data/resources/include_publicationStmt.xml">
+               <xi:fallback>
+                  <p>The Publications Statement should appear here. (c) Brown University</p>
+               </xi:fallback>
+            </xi:include>
+            <idno type="USEpigraphy" xml:id="IN.Bloom.IUAM.G.63.105.35">IN.Bloom.IUAM.G.63.105.35</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc>
+               <msIdentifier>
+                  <region>IN</region>
+                  <settlement>Bloom</settlement>
+                  <institution>IUAM</institution>
+                  <repository/>
+                  <idno>63.105.35</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="Demotic" otherLangs=""/>
```

Comment:
I'm not sure. Gaia and I thought it might be demotic because the book has [demoticum] written after the transcription they provide.

Reply:
I suspect what they're trying to suggest is that the next missing piece of the text is the demotic part of Lysikleides' name rather than indicate the language; given that this is from Attica, it's almost certainly just going to be in Attic Greek, so we'll call it "grc".